xwayland/xwm.c

```diff
@@ -1270,10 +1268,12 @@ static void xwm_get_render_format(struct wlr_xwm *xwm) {
 
 	if (format == NULL) {
 		wlr_log(L_DEBUG, "No 32 bit render format");
+		free(reply);
```

Comment:
Nitpick: add a NULL check for `reply` right after `xcb_render_query_pict_formats_reply`